src/add/operator/combineLatest.ts

```diff
@@ -5,6 +5,6 @@
 import {Observable} from '../../Observable';
 import {combineLatest} from '../../operator/combineLatest';
 
-Observable.prototype.combineLatest = combineLatest;
+Observable.prototype.combineLatest = <any>combineLatest;
```

Comment:
I've updated this in the last third commit as something separate from the other work. (Hence the larger number of files changed)